scripts/package.lua

```diff
@@ -40,7 +42,7 @@
 -- Make sure I've got what I've need to be happy.
 --
 
-	local required = { "git", "make", "gcc", "premake5", "zip" }
+	local required = { "git", "make", "gcc", "zip" }
```

Comment:
It is in the path, but in the case of the build scripts, I set the path to include it before calling premake5 package.
I guess what happens is the new shell environment isn't inheriting the current state of the environment, rather creating a new environment.